scipy/odr/models.py

```diff
@@ -108,6 +108,27 @@ def polynomial(order):
     polynomial : Model instance
         Model instance.
 
+    Examples
+    --------
+    We can fit an input data using orthogonal distance regression (ODR) with
+    a polynomial model:
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy import odr
+    >>> from scipy.odr.models import polynomial
+    >>> x = np.linspace(0.0, 5.0)
+    >>> y = np.sin(x)
+    >>> poly_model = polynomial(3)  # using third polynomial model
```

Comment:
minor: comment should be "using third order polynomial model"

Reply:
Thanks. I fixed it.